### DISCUSSION TITLE
通过容器编排之后无法在网站和数据库中使用

### DISCUSSION THREAD

**Discussion by yanzzchn** [1Panel-dev/1Panel#5071] [General]
因为我的开发环节和线上环节都是自己通过docker构建的镜像，然后通过docker-compose进行编排，今天第一次使用1panel，在容器里面通过我的docker-compose配置文件成功的将nginx、phpfpm、redis、mysql容器允许起来了。 但是发现了一个致命的问题，那就是在网站里面无法识别，还是让我去应用市场下载OpenResty,然后数据库里面也不认我的mysql和redis服务。也是让去下载。这样的话，那我使用1panel就没有意义了，和在命令行使用也没有区别了。我以为可以像传统那样在网站和数据库等地方进行图形化管理。不知道是我的使用方式有问题，还是产品的确是这样的。

**Comment by wan92hen** [comment]
1. Openresty 目前只能使用应用市场部署的，因为有一些跟面板功能相关的配置模板包含在里面；
2. 数据库的话需要自己在数据库管理页面，添加远程服务器的方式进行管理；

通过容器编排的方式运行应用，面板并不知道你运行的容器具体是什么，所以无法关联到特定的功能上。
